examples/3d/3d_generate_custom_mesh.rs

```diff
@@ -0,0 +1,245 @@
+// ! This example demonstrates how to create a custom mesh, and assign a custom UV mapping for a
+// ! cusotm texture. And how to change the UV mapping at run-time.
+use bevy::prelude::*;
+use bevy::render::mesh::Indices;
+use bevy::render::mesh::VertexAttributeValues;
+use bevy::render::render_resource::PrimitiveTopology;
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .add_systems(Startup, setup)
+        .add_systems(Update, rotate_y)
+        // .add_systems(Update, rotate_x)
+        // .add_systems(Update, rotate_z)
+        .add_systems(Update, input_handler)
+        .run();
+}
+
+// Rotate the cube to show off all the sides.
+#[allow(dead_code)]
```

Comment:
I think it's probably nicer to hook this up to user input rather than forcing them to comment in / out the code.

Maybe just X/Y/Z key presses to toggle each rotation on and off?

Reply:
Good idea